.travis.yml

```diff
@@ -1,17 +1,17 @@
 os: linux
 language: python
-dist: bionic
+dist: focal
```

Comment:
is it available on travis for all repo?

Reply:
Yes, but it's not official, I've only seen this used in Travis owned repos. See https://github.com/python-trio/trio/pull/1505 for more discussion about this